### DISCUSSION TITLE
covariate

### DISCUSSION THREAD

**Discussion by nskoch** [nlmixrdevelopment/nlmixr#582] [Q&A]
Hi,
is it possible to have the covariates in the dataset following a fit?
To plot EBE vs covariate.
Best regards
Nicolas SIMON

**Comment by mattfidler** [comment]
Hi Nicolas,

The covariates should be added to the dataset following the fit.  There may be a bug in saem or focei where it isn't added for that method.

Opened an issue here:

https://github.com/nlmixr2/nlmixr2/issues/20
